app/static/js/volunteerProfile.js

```diff
@@ -80,25 +80,27 @@ $(document).ready(function(){
              "endDate":$("#banEndDatepicker").val() //Expected to be a date in this format YYYY-MM-DD
             },
       success: function(response) {
-        
+
         location.reload();
       }
     });
   });
 
 
   $(".backgroundCheck").change(function () { // Updates the Background check of a volunteer in the database
+    checkType = $(this).attr("id")
     let data = {
-        checkPassed : $(this).val(), //Expected to be either a 0 or a 1
-        user: $(this).data("username"), //Expected to be the username of a volunteer in the database
-        bgType: $(this).attr("id") // Expected to be the ID of a background check in the database
+        checkPassed : $(this).val(),      // Expected to be either a 0 or a 1
+        user: $(this).data("username"),   // Expected to be the username of a volunteer in the database
+        bgType: checkType,       // Expected to be the ID of a background check in the database
+        bgDate: $("#" + checkType + "_date").val()  // Expected to be the type (ex. FBI) and  _date
```

Comment:
The comment should be about the date itself, not what the line is doing. like, `// The date of the background check completion in YYYY/MM/DD format` (or whatever the format is)